torch/csrc/autograd/python_function.cpp

```diff
@@ -82,7 +82,7 @@ auto PyFunction::legacy_apply(const variable_list& inputs) -> variable_list {
         msg += "')'";
         throw std::runtime_error(msg);
       }
-      tensor_results[i] = ((THPVariable*)obj)->cdata.data();
+      tensor_results[i] = ((THPVariable*)obj)->cdata;
```

Comment:
It's okay to remove `.data()` here, because the `wrap_outputs(...)` call below (implementation: https://github.com/pytorch/pytorch/blob/master/torch/csrc/autograd/functions/utils.cpp#L19 and https://github.com/pytorch/pytorch/blob/master/torch/csrc/autograd/functions/utils.cpp#L28) always calls `make_variable(...)` on each tensor in `outputs`, which ignores the tensor's original AutogradMeta and creates a new AutogradMeta.